### PR TITLE
Update PyPi Documentation

### PR DESCRIPTION
**Pull request by sspintel**

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -2,68 +2,77 @@
...
+- Intel<sup>®</sup> CPUs
+- Intel<sup>®</sup> integrated GPUs
+- Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs)
 
         pip3 install pip==21.0.1
         pip3 install tensorflow==2.5.1
         pip3 install -U openvino-tensorflow
 
-To use **OpenVINO™ integration with TensorFlow** with pre-installed OpenVINO™ binaries, please visit this page for detailed instructions: ([**OpenVINO™ integration with TensorFlow** - README](docs/BUILD.md))
+To leverage Intel® Vision Accelerator Design with Movidius™ (VAD-M) for inference, please refer to: [**OpenVINO™ integration with TensorFlow** alongside the Intel® Distribution of OpenVINO™ Toolkit](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/INSTALL.md#12-install-openvino-integration-with-tensorflow-alongside-the-intel-distribution-of-openvino-toolkit).
```
Comment (ck-intel) [inline python/README.md:31]: The Link in this line seems to be not working

File: python/README.md [hunk 2]
```diff
@@ -2,68 +2,77 @@
...
+
+For more details on installation please refer to [INSTALL.md](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/INSTALL.md), and for build from source options please refer to [BUILD.md](https://github.com/openvinotoolkit/openvino_tensorflow/blob/master/docs/BUILD.md)
```
Comment (ck-intel) [inline python/README.md:33]: IINSTALL.md link seems to be not working

Comment (sspintel) [reply python/README.md:33]: As the link points to master, and master doesn't have this file yet, it fails.

File: python/README.md (hunk 1 above)
Comment (sspintel) [reply python/README.md:31]: This file is supposed to reside on PyPi, where relative links are not possible. We have no option but to point them to specific files in specific branches.

**Review by ck-intel — APPROVED** [review]
Changes are ok

**Pull request merged**